front-end/src/components/Navigation.js

```diff
@@ -1,11 +1,16 @@
-import React from "react";
+
 import { AppBar, Toolbar, Link, Box } from "@mui/material";
 
+
 const pages = [
 	{ name: "HOME", link: "/" },
 	{ name: "COURSES", link: "/courses" },
+	
```

Comment:
Can you remove the additional space that you have on these lines. If you install the prettier extension, it will automatically remove those for you each time you save the file